deployment/src/strongmind_deployment/route53_zone.py

```diff
@@ -0,0 +1,77 @@
+from enum import Enum
+from typing import Optional, Dict
+import pulumi
+import pulumi_aws as aws
+from pydantic import BaseModel
+
+
+class Route53ZoneArgs(BaseModel):
+    domain_name: str
+    vpc_id: Optional[str] = None
+    vpc_region: Optional[str] = None
+    comment: Optional[str] = "Managed by Pulumi"
+    force_destroy: Optional[bool] = False
+    delegation_set_id: Optional[str] = None
+    tags: Optional[Dict[str, str]] = None
+
+
+class R53ZoneOutputs(str, Enum):
+    ROUTE53_ZONE_ID = "zone_id"
+    ROUTE53_ZONE_NAME = "zone_name"
+    ROUTE53_ZONE_ARN = "zone_arn"
+    ROUTE53_ZONE_DNS_SERVERS = "dns_servers"
+
+    def __str__(self):
+        return self.value
+
+
+class Route53Zone(pulumi.ComponentResource):
+    zone: aws.route53.Zone
+    domain_name: str
+
+    def __init__(self, name, args: Route53ZoneArgs, opts=None):
+        super().__init__("custom:Route53:Zone", name, {}, opts)
```

Comment:
resource naming again.  
There is precedence: 
https://github.com/StrongMind/public-reusable-workflows/blob/0f02af92aa42641b7271fd66a51dfa35836d99ff/deployment/src/strongmind_deployment/container.py#L37

Reply:
followed this